packages/cli/package.json

```diff
@@ -123,7 +123,8 @@
     "prepublishOnly": "pnpm run build"
   },
   "dependencies": {
-    "@prisma/engines": "workspace:*"
+    "@prisma/engines": "workspace:*",
+    "arg": "5.0.2"
```

Comment:
This needs to be moved under `devDependencies`, it will be bundled then

Reply:
But actually you should probably use 
```
import { arg } from '@prisma/internals'
```
instead

Reply:
I was, it was installed for the types, but I've since refactored anyways and relying directly on `argv` in the dispatch function now.